### DISCUSSION TITLE
GPT 4o mini support?

### DISCUSSION THREAD

**Discussion by semanticist21** [niieani/gpt-tokenizer#46] [General]
Gpt-4o-mini model is recently release.

Are there any plans for supporting this model?
I'm using it as gpt 4o currently.

**Comment by niieani** [comment]
I believe it mini uses the same tokenizer as regular 4o, which we already support.

**Reply by semanticist21** [reply]
> I believe it mini uses the same tokenizer as regular 4o.

Could you update the package with 4o type support? (If it is same with 4o)
Or can I make a PR.

**Reply by niieani** [reply]
4o type is already supported. You can make a PR to add 4o-mini as an alias though.
